types.go

```diff
@@ -394,7 +394,7 @@ func parseFAHDuration(s string) (time.Duration, error) {
 		days = daysTemp
 
 		if dIndex >= len(shortened)-1 { // s only contains days
-			return time.Hour * 24 * time.Duration(days), nil
+			return time.Duration(float64(time.Hour) * 24 * days).Truncate(time.Second), nil
```

Comment:
We don't need the `.Truncate(time.Second)` bit. Output is not this function's responsibility, plus we want to preserve as much information as possible.

Reply:
Good point, will remove.